mitodl/micromasters
# Configured sass and css in webpack

#### What are the relevant tickets?
Fixes #36 

#### What's this PR do?
Sets up CSS and Sass integration with webpack, and points the index view to the right place. It also breaks the webpack javascript into three chunks: style, inner, and outer (feel free to suggest better names)

#### Where should the reviewer start?
webpack.config.dev.js

#### How should this be manually tested?
Go to the index page, view source. The javascript links should be there and should be valid links. Also, if you add style information to static/scss/layout.scss, it should immediately show up in your browser due to the hot reloading
